ReportUtilityBills/resources/Data/UtilityRates/Natural Gas.csv

```diff
@@ -0,0 +1,69 @@
+Location,Undajusted,Household,Number of,Total
```

Comment:
Can we replace these fossil fuel CSVs with ones directly from EIA? For example, convert the second tab in the Download Series History link [here for natural gas](https://www.eia.gov/dnav/ng/ng_pri_sum_a_EPG0_PRS_DMcf_a.htm) to CSV and then use the latest available data (2021 or 2020, depending on state). Be sure to do unit conversions as needed.

Reply:
@shorowit Can you help me find rate CSVs for oil and propane?

Reply:
Fuel oil: https://www.eia.gov/dnav/pet/xls/PET_PRI_WFR_A_EPD2F_PRS_DPGAL_W.xls
Propane: https://www.eia.gov/dnav/pet/xls/PET_PRI_WFR_A_EPLLPA_PRS_DPGAL_W.xls

Note that the values are weekly (and only for the half of the year when there's heating). Rather than just using the latest available week, which might be a temporary spike, I suggest we average over the last 26 values (i.e., an entire heating season).

For states that aren't explicitly called out, you'll have to use the region values instead. Here are the regions: https://www.eia.gov/todayinenergy/detail.php?id=4890

Reply:
Fuel oil: shows PADD 1 - 2
Propane: shows PADD 1 - 4

Reply:
I guess we have to use the national value if no region is available. (Anytime we default to a region or national number for a given state, we should give a warning.)